src/Gaufrette/Adapter/Local.php

```diff
@@ -250,17 +249,13 @@ protected function ensureDirectoryExists($directory, $create = false)
      *
      * @param string $directory Path of the directory to create
      *
-     * @throws InvalidArgumentException if the directory already exists
-     * @throws RuntimeException         if the directory could not be created
+     * @throws \InvalidArgumentException if the directory already exists
```

Comment:
Maybe I miss something but the `\InvalidArgumentException` will never be thrown here.